shared/router-v2/account-switcher/index.tsx

```diff
@@ -58,6 +60,13 @@ type AccountRowProps = {
 }
 const AccountRow = (props: AccountRowProps) => {
   const [clicked, setClicked] = React.useState(false)
+  const prevWaiting = usePrevious(props.waiting)
+  React.useEffect(() => {
+    if (prevWaiting && !props.waiting) {
+      setClicked(false)
+    }
+  }, [setClicked, props.waiting, prevWaiting])
```

Comment:
`prevWaiting` doesn't seem necessary -- if `waiting` changes, it has to be different than previous one right? `setClicked` never changes.

Reply:
Good point.